lesson6.js

```diff
@@ -12,6 +18,7 @@ function makeLinkList() {
 }
 
 makeLinkList().then((item) => {
+      loading.remove();
```

Comment:
Line 6, `const loading = document.getElementById('loading-img');` should be declared here.